### PR TITLE
feat: slice register ui

### PR DESCRIPTION
**Pull request by jeeehaan**
url dev:
http://localhost:3000/register

ui:
image

### CODE REVIEW

**Comment by fatjan** [comment]
> url dev: http://localhost:3000/register
> 
> ui: image

Ini page nya register atau login?

Judul sama button beda nama

File: jest.config.mjs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import nextJest from 'next/jest.js';
+
+const createJestConfig = nextJest({
+  // Provide the path to your Next.js app to load next.config.js and .env files in your test environment
```
Comment (fatjan) [inline jest.config.mjs]: I think we can remove these unnecessary comments.

File: src/components/(auth)/auth/register.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+export const Register = () => {
+  return (
+    <main className="flex h-screen items-center justify-center">
+      <div className="flex w-[300px] flex-col space-y-3">
+        <h3>Register</h3>
+        <p>Sign up to Indonesia Open Course</p>
+        <Input type="text" placeholder="Enter your name" />
+        <Input type="email" placeholder="Enter your email" />
+        <Input type="password" placeholder="Enter your password" />
+        <Button>Login</Button>
```
Comment (fatjan) [inline src/components/(auth)/auth/register.tsx]: The button should probably be a register button?

Or does it mean once this button is clicked, the user will do register and login at the same time?

**Review by fatjan — CHANGES_REQUESTED** [review]
Kindly update as per comments/suggestions. Thanks

File: src/components/(auth)/auth/register.tsx (hunk 2 above)
Comment (jeeehaan) [reply src/components/(auth)/auth/register.tsx]: should be register button, my mistake

File: jest.config.mjs (hunk 1 above)
Comment (jeeehaan) [reply jest.config.mjs]: will do

**Comment by jeeehaan** [comment]
updated @fatjan

**Review by fatjan — APPROVED** [review]

**Review by ahmadrasyidsalim — APPROVED** [review]

**Pull request merged**